### PR TITLE
Fix Bug 1432672 - Remove sidebar preferences and have gear open about:preferences

### PR DESCRIPTION
**Pull request by sarracini**
1. Removes sidebar 
2. Makes gear open about:preferences

### CODE REVIEW

File: locales/en-US/strings.properties [hunk 1]
```diff
@@ -88,25 +88,6 @@ prefs_topstories_sponsored_learn_more=Learn more
 prefs_highlights_description=A selection of sites that you’ve saved or visited
 prefs_snippets_description=Updates from Mozilla and Firefox
 settings_pane_button_label=Customize your New Tab page
-settings_pane_header=New Tab Preferences
```
Comment (Mardak) [inline locales/en-US/strings.properties]: These strings can't be removed yet as we will likely uplift strings to beta

Comment (Mardak) [reply locales/en-US/strings.properties]: Post a comment to https://bugzilla.mozilla.org/show_bug.cgi?id=1445090 with these string ids that should be removed.

Comment (sarracini) [reply locales/en-US/strings.properties]: How about pocket_description, settings_pane_topstories_options_sponsored, settings_pane_highlights_body2 based on Bug 1404890? Can I remove those?

**Comment by rlr** [comment]
Basically, any strings that are used in 60 need to stay another ~6-8 weeks until after the last string export to beta. :+1:

**Comment by as-pine-proxy** [comment]
[Treeherder: 228d9bc2d46e1b87c2d7d64425f3df613a4a1aa4](https://treeherder.mozilla.org/#/jobs?repo=pine&revision=228d9bc2d46e1b87c2d7d64425f3df613a4a1aa4)

**Pull request merged**